docs/codeql/codeql-cli/creating-codeql-databases.rst

```diff
@@ -208,14 +208,9 @@ commands that you can specify for compiled languages.
 
      codeql database create cpp-database --language=cpp --command=make
 
-- C# project built using ``dotnet build`` (.NET Core 3.0 or later)::
+- C# project built using ``dotnet build``::
```

Comment:
If the extra options `p:UseSharedCompilation=false /t:rebuild` are always required for projects built using dotnet, then I guess you might want to call that out in the text as well as the example command.

Also which version of the CLI was this changed introduced? If earlier versions of the CLI require different behaviour then that should be mentioned here too.

Reply:
Added. This was not a change that got introduced, but a bug that has been there all along. We are working on a long term solution that will inject the `/p:UseSharedCompilation=false` flag, but until we have that it has to be added manually.

Reply:
I see. Thanks for the explanation.